tests/test_new_excerpt.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import time
+import requests
+from new_excerpt import new_excerpt, click_and_check
+from selenium import webdriver
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium.common.exceptions import NoSuchElementException
+from selenium.webdriver.support import expected_conditions as EC
```

Comment:
`ALL_CAPS` names are reserved for constants. If you need a short-cut, better directly
```python
from selenium.webdriver.support.expected_conditions import presence_of_element_located
```
to import `presence_of_element_located` into the module (= Python file) scope, as `presence_of_element_located` seems to be the only thing you use from `expected_conditions`.